pkg/sqlite/gallery.go

```diff
@@ -102,7 +116,18 @@ func (r *galleryRowRecord) fromPartial(o models.GalleryPartial) {
 	r.setNullString("url", o.URL)
 	r.setSQLiteDate("date", o.Date)
 	r.setNullString("details", o.Details)
-	r.setNullInt("rating", o.Rating)
+
+	// prefer rating100 over rating	
+	if o.Rating100.Set {
+		r.setNullInt("rating", o.Rating100)
+	} else if o.Rating.Set {
+		v := o.Rating
+		if v.Value != 0 {
+			v.Value = models.Rating5To100(v.Value)
+		}
+		r.setNullInt("rating", v)
+	}
```

Comment:
This should be able to be reverted. `Rating` should already be expressed as 1-100.